src/main/java/pl/Lukasz/Calories/encrypting.java

```diff
@@ -0,0 +1,13 @@
+package pl.Lukasz.Calories;
+
+
+import org.springframework.security.crypto.bcrypt.BCrypt;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+public class encrypting {
```

Comment:
Classes always use pascal case - this is a test class - you should never commit a code that you don't want to see or share and it's only useful for your local testing.